sdk/python/endpoints/online/mlflow/online-endpoints-deploy-mlflow-model.ipynb

```diff
@@ -2,43 +2,40 @@
  "cells": [
```

Comment:
What is this file and why is it different than the other one you're adding in this PR?

Reply:
Assuming that it's just the difference between a custom deploy and NCD, maybe the notebook names should be more descriptive of that?

Reply:
Are you suggesting to name this one NCD? I don't one to use that name that much cause it is an internal way to name the deployment. That's why I named the other one "with-script" to indicate it is with an scoring script. Should I name it custom?

Reply:
you could name it -no-script directly if that would be helpful for users